test/extended/setup.sh

```diff
@@ -2,6 +2,8 @@
 #
 # This abstracts starting up an extended server.
 
+source "${OS_ROOT}/hack/util.sh"
```

Comment:
@stevekuznetsov I've got some reviews on my plate for the rest of the day. If you wanna peak, you're welcome. Otherwise, I'll debug tomorrow.